metabrainz/admin/views.py

```diff
@@ -53,8 +53,12 @@ def details_stats(self, user_id):
     @expose('/approve')
     def approve(self):
         user_id = request.args.get('user_id')
-        User.get(id=user_id).set_state(STATE_ACTIVE)
-        flash.info("User #%s has been approved." % user_id)
+        if request.args.get('limited'):
+            User.get(id=user_id).set_state(STATE_LIMITED)
+            flash.warn("User #%s has been approved (with limited access)." % user_id)
```

Comment:
Please use the same message for both STATE_LIMITED and STATE_ACTIVE. ("User #%s has been approved.")